test/common/README.md

```diff
@@ -243,6 +243,11 @@ Platform check for Windows.
 
 Platform check for Windows 32-bit on Windows 64-bit.
 
+### isSymbolAvailable
```

Comment:
Can we use a more descriptive name than this please?

Reply:
my bad,it should be `isCPPSymbolNotMappedOS`